core/inventory.go

```diff
@@ -18,8 +19,8 @@ var (
 
 // InventoryListing is the listing representation stored on IPFS
 type InventoryListing struct {
-	Inventory   int64  `json:"inventory"`
-	LastUpdated string `json:"lastUpdated"`
+	Inventory   *big.Int `json:"inventory"`
```

Comment:
This is not going to marshal to JSON in the way we expect. By default big.Int marshals to int and we probably would rather handle them as strings. https://play.golang.org/p/RvpU9y8wlBD